### DISCUSSION TITLE
Overwhelming releases binaries to pick from without explanation

### DISCUSSION THREAD

**Discussion by inuonly** [VSCodium/vscodium#1543] [Ideas]
hi , 
Steps to reproduce - 
we were trying to install vscodium on one of our windows system so we started our journey from - 
1. https://vscodium.com/ as search engine gives that result with vscodium as keyword. 
2. there was release option at start or a big button to click sayaing download lates release which leads to https://github.com/VSCodium/vscodium/releases
3. the list is shor so we need to click show all in the assets only to find so many names of binary releases for various platforms
4. we arrive at the problem which says lot of binaries, now we could not find any description (readme) nor could we find any documentation in the installation section nor the wiki page had any information to help us select. 

Perspective : Ofcourse while installing we know exe works on windows and deb works on ubuntu based linux etc etc but then there are signatures , then 32 bit, 64 bit, then above all user setup , normla setup named files, and no place to know what or which one does what in your system . 
Suggestion:
So a guide telling which version is needed for what purpose since you are posting 116 files in release, considering that , this information seems very helpful. 

PS: If we are missing some info or details please let us know and feel free to discard this suggesiton. Also since landing page is readme in github so that will be nice place apart from wiki page for such instructions.

TLDR : which one to chose from is not mentioned anywhere in documentation that we could find nor in the Readme which is normal place.
